### PR TITLE
Adoption for rsa pack

### PR DESCRIPTION
**Pull request by adit-ghildiyal**
PR to complete adoption process

### CODE REVIEW

**Review by edik24 — DISMISSED** [review]
Please update the readme file with a description about the pack and what it does, similar to the pack in the other [PR](https://github.com/demisto/content/pull/34140)

**Comment by adit-ghildiyal** [comment]
Hi @maimorag ,
I will be updating the readme file in a while, after that the PR is ready for review.

**Comment by adit-ghildiyal** [comment]
Hi @maimorag @edik24 
I have updated the Readme file. 
Also for this pack I will be raising another MR in sometime for some command updates for this pack. The last MR which I raised for the same was closed because I could not upload the demo video.
Until that time this adoption MR can be reviewed and merged.

File: Packs/RsaNetWitnessPacketsAndLogs/ReleaseNotes/1_0_8.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Completed adoption process.
```
Comment (maimorag) [inline Packs/RsaNetWitnessPacketsAndLogs/ReleaseNotes/1_0_8.md:1]:
```suggestion

## RSA NetWitness

- Completed adoption process.

```

**Review by maimorag — COMMENTED** [review]
Hey @adit-ghildiyal ,
Thank you for your contribution!
Good work :)

**Review by maimorag — APPROVED** [review]

**Pull request merged**